src/main/webapp/resources/js/apis/passwordReset.js

```diff
@@ -23,7 +23,43 @@ export const passwordResetApi = createApi({
       }),
       invalidatesTags: ["PasswordReset"],
     }),
+    createPasswordResetEmail: build.mutation({
```

Comment:
Can we rename this file to `password-reset.js`  trying to get rid of camelcase js and html files / routs.  They are just ugly.

Reply:
Renamed in 66fd23c47ca7b634ae1918ac77341a2fdf53048e